src/main/resources/constants/identifiers.json

```diff
@@ -1,11 +1,13 @@
 {
   "id-type:DOI": {
     "title": "doi",
-    "viewName": "doi"
+    "viewName": "doi",
+    "baseUrl": "https://doi.org/"
   },
   "id-type:URN": {
     "title": "urn",
-    "viewName": "urn"
+    "viewName": "urn",
+    "baseUrl": "http://persistent-identifier.nl/"
```

Comment:
this holds only for the dutch URN-NBN, this field is for any URN, which is broader

Reply:
Then we got a problem in here: https://github.com/DANS-KNAW/easy-deposit-api/blob/30f196bf9d32e10e14981007531f21e333ff69fe/src/main/scala/nl.knaw.dans.easy.deposit/docs/DDM.scala#L109

Probably we should fix that one first if this URN field is not just for our URN-NBN...

Reply:
we decided to make it more explicit to make `viewName: urn:nbn:nl` and add format checking to URN (`^urn:nbn:nl:ui:.*$`) and DOI (`^(http://dx.doi.org/|https://doi.org/){0,1}10.(\d)+(\.(\d)+)*/.*$`)